v1/content.yaml

```diff
@@ -366,9 +366,18 @@ paths:
         - application/json
       responses:
         '200':
-          description: The existing revision of the page matches the sent text
+          description: The content was not changed, and no new version was created.
+          schema:
+            type: string
```

Comment:
Based on https://github.com/wikimedia/restbase/blob/master/sys/page_save.js#L185 and saveWikitext further up I'm pretty sure that this is returning the (massaged) PHP API response, rather than a string.